openedx/openedx-k8s-harmony
# Add __init__.py to harmony_search

Without this, the module does not get installed correctly when installing the package with pip.

**Test instructions**:

1. Create a fresh virtual environment.
2. Install harmony plugin without this patch: `pip install "git+https://github.com/openedx/openedx-k8s-harmony.git@harmony-chart-0.5.1#egg=tutor-contrib-harmony-plugin&subdirectory=tutor-contrib-harmony-plugin"`
3. Start the python interpreter and try to import the `harmony_search` module: `from tutor_k8s_harmony_plugin.harmony_search.base import BaseSearchAPI`; you will get an import error
4. Uninstall the plugin and re-install a version with this patch: `pip uninstall -y tutor-contrib-harmony-plugin && pip install "git+https://github.com/open-craft/openedx-k8s-harmony.git@mtyaka/add-init-files#egg=tutor-contrib-harmony-plugin&subdirectory=tutor-contrib-harmony-plugin"`
5. Run the python interpreter again and verify that you can successfully import the module this time: `from tutor_k8s_harmony_plugin.harmony_search.base import BaseSearchAPI`
